packages/multi-select-combo-box/src/vaadin-multi-select-combo-box-container.js

```diff
@@ -50,6 +55,11 @@ class MultiSelectComboBoxContainer extends InputContainer {
 
```

Comment:
**nit:** I guess the wrapper can now be added with a simple `appendChild`, as `slots[2]` is moved to its own div further below.

Reply:
Reverted the DOM change.